articles/hooks/guides/edit-hooks-using-cli.md

```diff
@@ -6,18 +6,30 @@ topics:
     - cli
 contentType: how-to
 useCase: extensibility-hooks
+v2: true
 ---
+# Edit Existing Hooks Using the Command-Line Interface
 
-# Edit Existing Hooks
+The Auth0 Command-Line Interface (CLI) allows you to edit existing Hooks associated with specific extensibility points within the Auth0 platform.
 
 ::: warning
 Tenants created after **July 16, 2018** will not have access to the underlying Webtask Sandbox via the Webtask CLI. Please contact [Auth0](https://auth0.com/?contact=true) to request access.
 :::
 
+## Set up the CLI
+
+You can find instructions for installing and configuring the Webtask CLI in the [Dashboard > Webtask page](${manage_url}/#/account/webtasks). 
+
+The `wt-cli` package also includes the `auth0` binary, allowing you to use the Auth0 CLI.
+
+![Install Webtasks Instructions](/media/articles/hooks/mgmt-dashboard-webtasks.png)
+
 ::: note
 All of the examples on this page use `auth0-profile` as the name of the profile. This is the same profile name used when installing `wt-cli` and can be obtained from *Step 2* of the instructions set located on [Auth0 Management Dashboard's Webtask page](${manage_url}/#/account/webtasks).
 :::
 
+## Edit your Hook
+
 You can edit the code of your Hook using the [Webtask Editor](https://webtask.io/docs/editor). The following command will open up the code for your Hook in the Webtask Editor window:
```

Comment:
You don't need "You can" at the beginning. I naturally write that way and then go back and self-edit to remove as many as possible to make the text more active voice.